jsdocConfig.json

```diff
@@ -0,0 +1,23 @@
+{
+  "tags": {
+    "allowUnknownTags": true
+  },
+  "opts": {
+    "destination": "./docs"
```

Comment:
I'm fine with using the same patterns as the other repos. This process works fine...